src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -497,7 +499,9 @@ namespace winrt::Microsoft::Terminal::Control::implementation
             }
             else if (auto solidColor = RootGrid().Background().try_as<Media::SolidColorBrush>())
             {
+                const auto originalOpacity = solidColor.Opacity();

```

Comment:
Because .Color will erase it?

Reply:
That's my guess. I didn't experiment too carefully with this though.